src/pages/event/[id].tsx

```diff
@@ -41,16 +45,18 @@ const EventInfoHeader: React.FC = () => {
 
   const [isDeleteDialogOpen, setIsDeleteDialogOpen] = useState(false);
   const [isEditDialogOpen, setIsEditDialogOpen] = useState(false);
+  const [isLeaveDialogOpen, setIsLeaveDialogOpen] = useState(false);
 
   const editEventName = api.events.updateEvent_name.useMutation();
   const deleteEvent = api.events.deleteEvent.useMutation();
+  const deleteParticipate = api.participates.deleteParticipate.useMutation();
   const editEventParticipants =
```

Comment:
Does `deleteParticipate` remove all the time slots as well?

Reply:
I do not think so.